problems/3.conditionals.js

```diff
@@ -1,30 +1,56 @@
 /** Returns "It's freezing!" if the temperature is below 32, and "It's alright" otherwise
-* @param {number} temp - The current temperature
-* @returns {string} - A message about the temperature
-*/
-function temperatureMessage() {
-
+ * @param {number} temp - The current temperature
+ * @returns {string} - A message about the temperature
+ */
+function temperatureMessage(temp) {
+    let num = -Infinity < 32
```

Comment:
Why check if -Infinity is less than 32? Won't that always be true? 

Instead we need to be checking if temp < 32

Reply:
this where i need to better understand the problem. Like today I see how you where able to approach and knowing that we have ceratin criteria to work with I love the fact that I saw this class today because it able me to now get this a step better.